portfolio/src/main/webapp/index.html

```diff
@@ -7,11 +7,15 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComment()">
     <div id="header">
         <div class="headBox"> <a href="index.html"> Home </a> </div>
     </div>
 
+    <div id="comments-container">
+        
+    </div>
```

Comment:
if this is just a place holder, you can remove the space
```suggestion
    <div id="comments-container"></div>
```